internal/logger/logger.go

```diff
@@ -0,0 +1,68 @@
+package logger
+
+import (
+	"fmt"
+	"os"
+
+	"go.opentelemetry.io/otel/otellog"
+)
+
+var Logger otellog.Logger = otellog.NewDefaultLogger(os.Stdout, otellog.LogLevelInfo)
```

Comment:
I think we need to provide a mechanism somewhere for the application author to provide an alternate implementation.

Reply:
Agreed. I would like it if this closer followed what we do with the [`ErrorHandler` interface](https://github.com/open-telemetry/opentelemetry-go/blob/main/handler.go).

Reply:
yes I was thinking to add `SetLogger` API to add custom logging implementation but I will try to follow @MrAlias's suggestion and lookup `ErrorHandler` interface to maintain implementation parity within SDK.

Reply:
What I was suggesting, in a very terse manner, was that we include a `SetLogger` and `GetLogger` function in the `otel` package and move the `Logger` interface to the `otel` package as well. This would mirror what is done with the `ErrorHandler`. From there, the default implementation could be abstracted away into an internal package.